src/qt/optionsmodel.cpp

```diff
@@ -80,6 +80,11 @@ void OptionsModel::Init(bool resetSettings)
         settings.setValue("fCoinControlFeatures", false);
     fCoinControlFeatures = settings.value("fCoinControlFeatures", false).toBool();
 
+    if (!settings.contains("m_make_unsigned_button")) {
```

Comment:
I'm not sure if we have a standard naming schema for QSettings at this point, but `m_` seems wrong at least? (Since they're typeless, keeping the hungarian notation makes sense...)

Reply:
Done